sessions/006/README.md

```diff
@@ -551,3 +551,343 @@ pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b   20Gi       RWO            Delete
+
+$ NODE_HOSTNAME_0="$(kubectl describe pod my-cluster-zookeeper-0 | grep Node: | awk '{print $2}' | cut -d / -f1)"
+$ NODE_HOSTNAME_1="$(kubectl describe pod my-cluster-zookeeper-1 | grep Node: | awk '{print $2}' | cut -d / -f1)"
+
+$ kubectl run kubectl-remove-zookeeper-0 -itq --rm --restart "Never" --image "foo" --overrides "$(cat sessions/006/resources/patch-zk.yaml \
+| yq ".spec.nodeSelector.\"kubernetes.io/hostname\" = \"$NODE_HOSTNAME_0\"" \
+| yq ".spec.volumes[0].persistentVolumeClaim.claimName = \"data-$CLUSTER_NAME-zookeeper-0\"" \
+| yq -p yaml -o json)"
+removed '/zookeeper/data/version-2/acceptedEpoch'
+removed '/zookeeper/data/version-2/currentEpoch'
+removed '/zookeeper/data/version-2/log.200000001'
+removed '/zookeeper/data/version-2/snapshot.0'
+
+$ kubectl run kubectl-remove-zookeeper-1 -itq --rm --restart "Never" --image "foo" --overrides "$(cat sessions/006/resources/patch-zk.yaml \
+| yq ".spec.nodeSelector.\"kubernetes.io/hostname\" = \"$NODE_HOSTNAME_1\"" \
+| yq ".spec.volumes[0].persistentVolumeClaim.claimName = \"data-$CLUSTER_NAME-zookeeper-1\"" \
+| yq -p yaml -o json)"
+removed '/zookeeper/data/version-2/acceptedEpoch'
+removed '/zookeeper/data/version-2/currentEpoch'
+removed '/zookeeper/data/version-2/log.200000001'
+removed '/zookeeper/data/version-2/snapshot.0'
+```
+
+Restart all failed pods so that they can sync up with the leader.
+
+```sh
+$ kubectl delete po $CLUSTER_NAME-zookeeper-0 --force && kubectl delete po $CLUSTER_NAME-zookeeper-1 --force
+pod "my-cluster-zookeeper-0" deleted
+pod "my-cluster-zookeeper-1" deleted
+
+$ kubectl get po -l strimzi.io/cluster=$CLUSTER_NAME
+NAME                                          READY   STATUS    RESTARTS   AGE
+my-cluster-entity-operator-7d6d6bd454-sh7rc   3/3     Running   0          96s
+my-cluster-kafka-0                            1/1     Running   0          5m9s
+my-cluster-kafka-1                            1/1     Running   0          5m9s
+my-cluster-kafka-2                            1/1     Running   0          5m9s
+my-cluster-zookeeper-0                        1/1     Running   0          6m14s
+my-cluster-zookeeper-1                        1/1     Running   0          6m14s
+my-cluster-zookeeper-2                        1/1     Running   0          6m14s
+
+$ kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --create --topic t0
+Created topic t0.
+
+$ kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --describe --topic t0
+Topic: t0	TopicId: 1Sgy6V-CR0K7MMJ9khDuRw	PartitionCount: 3	ReplicationFactor: 3	Configs: min.insync.replicas=2,message.format.version=3.0-IV1
+	Topic: t0	Partition: 0	Leader: 0	Replicas: 0,2,1	Isr: 0,2,1
+	Topic: t0	Partition: 1	Leader: 2	Replicas: 2,1,0	Isr: 2,1,0
+	Topic: t0	Partition: 2	Leader: 1	Replicas: 1,0,2	Isr: 1,0,2
+```
+
+If there are only 1 node with storage issue, please go through the same process for the specific node.
```

Comment:
Should we remove this line?

Reply:
As discussed, we can keep this line since it will let users know this procedure also works for single node down case.